server/parser/src/main/antlr4/org/eclipse/lsp/cobol/core/CobolParser.g4

```diff
@@ -2181,18 +2173,20 @@ length
    ;
 
 argument
-   : arithmeticExpression
-   | TRAILING | LEADING
+   : ALL
```

Comment:
Looks like ALL is a valid argument as per  https://www.ibm.com/docs/en/cobol-zos/6.4?topic=reference-references-data-division-names and https://www.ibm.com/docs/en/cobol-zos/6.4?topic=reference-subscripting
But could not get to a compliable source which uses 'ALL' in table reference . Doc does say `Must be used only when the subscripted identifier is used as an intrinsic function argument or to identify a table in a format 2 SORT statement (table SORT statement).` but I couldn't figure out is meaning

Reply:
The ALL keyword can be used with some functions like SUM and AVERAGE when used on a table. The ALL keyword then means to use all of the the fields in the table for that specific dimension.

For example:
01 TABLE.
     05 NUMBERS-TABLE OCCURS 10 TIMES.
          10 NUMBER PIC 9(5).


....

COMPUTE SUMM = FUNCTION SUM(NUMBERS-TABLE(ALL)

This would sum NUMBER(1) to NUMBER(10).